docs/diagnostics/overview.md

```diff
@@ -3,204 +3,17 @@ title: .NET Aspire diagnostics overview
-
-To suppress the `ASPIREACADOMAINS001` diagnostic, select the **Copy** button from the following diagnostic ID, and see [Suppress diagnostic](#suppress-diagnostic):
-
-```plaintext
-ASPIREACADOMAINS001
-```
-
-## ASPIREHOSTINGPYTHON001
-
-<span id="ASPIREHOSTINGPYTHON001"></span>
-
-.NET Aspire provides a way to add Python executables or applications to the .NET Aspire app host. Since the shape of this API is expected to change in the future, it's experimental (<xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>). To suppress the compiler error/warning, use the following code:
-
-To suppress the `ASPIREHOSTINGPYTHON001` diagnostic, select the **Copy** button from the following diagnostic ID, and see [Suppress diagnostic](#suppress-diagnostic):
-
-```plaintext
-ASPIREAZURE001
-```
-
-## ASPIRECOSMOSDB001
-
-<span id="ASPIRECOSMOSDB001"></span>
-
-.NET Aspire provides a way to use the Cosmos DB Linux-based (preview) emulator. Since this emulator is in preview and the shape of this API is expected to change in the future, it's experimental (<xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>). To suppress the compiler error/warning, use the following code:
-
-To suppress the `ASPIRECOSMOSDB001` diagnostic, select the **Copy** button from the following diagnostic ID, and see [Suppress diagnostic](#suppress-diagnostic):
-
-```plaintext
-ASPIRECOSMOSDB001
-```
-
-## ASPIREPUBLISHERS001
-
-<span id="ASPIREPUBLISHERS001"></span>
-
-.NET Aspire introduced the concept of _Publishers_ starting in version 9.2. Publishers play a pivotal role in the deployment process, enabling the transformation of your distributed app into deployable assets. This alleviates the intermediate step of producing the publishing [manifest](../deployment/manifest-format.md) for tools to act on, instead empowering the developer to express their intent directly in C#.
-
-Publishers are currently in preview and the APIs are experimental (<xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>). To suppress the compiler error/warning, use the following code:
-
-To suppress the `ASPIREPUBLISHERS001` diagnostic, select the **Copy** button from the following diagnostic ID, and see [Suppress diagnostic](#suppress-diagnostic):
-
-```plaintext
-ASPIREPUBLISHERS001
-```
-
-## ASPIREAZURE001
-
-<span id="ASPIREAZURE001"></span>
-
-The .NET Aspire Azure hosting integration now ships with a publisher. If you're using any of the <xref:Aspire.Hosting.AzurePublisherExtensions.AddAzurePublisher*> APIs, you might see a compiler error/warning indicating that the API is experimental. This is expected behavior, as the API is still in preview and the shape of this API is expected to change in the future.
-
-To suppress the `ASPIREAZURE001` diagnostic, select the **Copy** button from the following diagnostic ID, and see [Suppress diagnostic](#suppress-diagnostic):
-
-```plaintext
-ASPIREAZURE001
-```
-
-## Suppress diagnostic
```

Comment:
Where does all of this content move to? Maybe just create a new file with this under the Overview and put this suppression stuff in there?

Reply:
I copied it out and I'll update the .NET docs with the information.